frontend/src/hooks/useTeamUtils.tsx

```diff
@@ -34,6 +37,9 @@ const useTeamUtils = (): TeamUtilsType => {
 
   const { teamId } = router.query;
 
+  const teamsList = useRecoilValue(teamsListState);
+  const setTeamsList = useSetRecoilState(teamsListState);
```

Comment:
```suggestion
  const [teamsList,setTeamsList] = useRecoilState(teamsListState);
```
You can also change the line 35 and 36 (membersList) to follow the same approach